src/libraries/System.Drawing.Primitives/src/System/Drawing/KnownColorTable.cs

```diff
@@ -441,28 +413,60 @@ internal static class KnownColorTable
             KnownColorKindWeb,      // WhiteSmoke
             KnownColorKindWeb,      // Yellow
             KnownColorKindWeb,      // YellowGreen
-#if FEATURE_WINDOWS_SYSTEM_COLORS
-            // "System" colors, Part 2
-            KnownColorKindSystem,       // ButtonFace
-            KnownColorKindSystem,       // ButtonHighlight
-            KnownColorKindSystem,       // ButtonShadow
-            KnownColorKindSystem,       // GradientActiveCaption
-            KnownColorKindSystem,       // GradientInactiveCaption
-            KnownColorKindSystem,       // MenuBar
-            KnownColorKindSystem,       // MenuHighlight
-#else
+
+            // "System" colors, Part 1
             KnownColorKindSystem,       // ButtonFace
             KnownColorKindSystem,       // ButtonHighlight
             KnownColorKindSystem,       // ButtonShadow
             KnownColorKindSystem,       // GradientActiveCaption
             KnownColorKindSystem,       // GradientInactiveCaption
             KnownColorKindSystem,       // MenuBar
             KnownColorKindSystem,       // MenuHighlight
-#endif
+
             // "Web" colors, Part 2
             KnownColorKindWeb,      // RebeccaPurple
         ];
 
+        private static ReadOnlySpan<uint> AlternateSystemColors =>
+        [
+            0,          // To align with KnownColor.ActiveBorder = 1
+
+                        // Existing   New
+            0xFF464646, // FFB4B4B4 - FF464646: ActiveBorder - Dark gray
```

Comment:
I trust you have some scheme for defining these?  Can you document it?

Reply:
Added some comments